shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -991,4 +991,11 @@ object GlutenConfig {
       .doc("When true, enable fallback reporter rule to print fallback reason")
       .booleanConf
       .createWithDefault(true)
+
+  // TODO, enable it by default after we fix velox write schema issue.
+  val COLUMNAR_PARQUET_WRITE_ENABLED =
```

Comment:
@JkSelf there is an issue about column name, so I disable this conversion by default now.

```sql
spark.sql("set spark.sql.optimizer.excludedRules=org.apache.spark.sql.catalyst.optimizer.ConvertToLocalRelation")
val df = Seq(true, false, true, true, false, false).toDF("a")
df.write.mode("overwrite").parquet("test")
spark.read.parquet("test").schema

// column name is n1_1
```

It seems the schema in columnar batch is not consistent with plan.

Reply:
The issue is not introduced by this pr. Disable it by default is for passing CI..

Reply:
Fixed in https://github.com/oap-project/gluten/pull/2395

Reply:
@JkSelf  thank you for the quick fix !